dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumResourceScope.kt

```diff
@@ -267,6 +275,9 @@ internal class RumResourceScope(
 
         val context = getRumContext()
         val user = CoreFeature.userInfoProvider.getUserInfo()
+        val hasUserInfo = user.id != null || user.name != null ||
```

Comment:
should we maybe extract it into extension/util method? the same check occurs multiple times in this PR